### PR TITLE
DDF-04483 fixing flickering user-notifications

### PR DESCRIPTION
**Pull request by gjvera**
#### What does this PR do?
As a result of #4484, user notifications flicker upon deletion. 
#### Select relevant component teams: 
@codice/ui 
#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler 
@djblue 
#### How should this be tested?
perform some actions that create notifications (e.g upload data)
ensure no abnormalities or regressions in the user-notification component

#### What are the relevant tickets?
For Jira:
[](https://codice.atlassian.net/browse/)

For GH Issues:
Fixes: #4483 

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js [hunk 1]
```diff
@@ -25,10 +25,9 @@ const Empty = styled.div`
   transition: transform ${props => props.theme.coreTransitionTime} linear;
   transform: scale(1);
   text-align: center;
-  font-size: ${props => props.theme.largeFontSize}
-  padding: ${props => props.theme.mediumSpacing}
+  font-size: ${props => props.theme.largeFontSize};
```
Comment (mojogitoverhere) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js]: @andrewkfiedler 	@djblue 	Is there a style rule we can add to catch issues like this?

Comment (andrewkfiedler) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js]: There's supposed to be one, not sure why it isn't catching these sorts of things.

**Comment by gjvera** [comment]
build now

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js (hunk 1 above)
Comment (vinamartin) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js]: Should that rule be fixed as part of this change?

**Comment by rymach** [comment]
before and after example videos provided by gabriel
[video.zip](https://github.com/codice/ddf/files/3009406/video.zip)

**Comment by rymach** [comment]
🍾 Hero Successful 👍

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js (hunk 1 above)
Comment (andrewkfiedler) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/user-notifications/user-notifications.js]: No, but I think we should open an issue and look into it.

**Pull request merged**